src/solver/conv_MP_bidirectional_winograd.cpp

```diff
@@ -530,5 +623,193 @@ template struct ConvMPBidirectWinograd<4, 3>;
+        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F6X3{}))
+            return false;
+    if(wino_data_tile == 5 && wino_filter_tile == 3)
+        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F5X3{}))
+            return false;
+    if(wino_data_tile == 4 && wino_filter_tile == 3)
+        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F4X3{}))
+            return false;
+    if(wino_data_tile == 3 && wino_filter_tile == 3)
+        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F3X3{}))
+            return false;
+
+    return IsApplicableTransform<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>(ctx) &&
+           ConvHipImplicitGemmForwardV4R4Xdlops().IsApplicable(GetTransformedConvContext(ctx));
+}
+
+template <int WinoDataH, int WinoFilterH, int WinoDataW, int WinoFilterW>
+ConvSolution
+ConvMPBidirectWinograd_xdlops<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::GetSolution(
+    const ConvolutionContext& ctx,
+    const PerformanceImplicitGemmForwardV4R4Xdlops& config,
+    bool) const
+{
+    ConvSolution wino_transform =
+        ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>{}.GetSolution(ctx);
+
+    const ConvolutionContext xdlops_conv_ctx = GetTransformedConvContext(ctx);
+
+    ConvSolution xdlops_conv =
+        ConvHipImplicitGemmForwardV4R4Xdlops{}.GetSolution(xdlops_conv_ctx, config);
+
+    ConvSolution result;
+    result.workspce_sz = wino_transform.workspce_sz + xdlops_conv.workspce_sz;
+
+    assert(xdlops_conv.construction_params.size() == 1);
+
+    // change transform layout
+    // GCNHW -> GNCHW
+    std::ostringstream additional_options_wei;
+    GenerateClangDefsym(additional_options_wei, "XDLOPS_CONV_GEMM", 1);
+    wino_transform.construction_params[1].comp_options += additional_options_wei.str();
+
+    result.construction_params.push_back(wino_transform.construction_params[0]);
+    result.construction_params.push_back(wino_transform.construction_params[1]);
+    result.construction_params.push_back(wino_transform.construction_params[2]);
+    result.construction_params.push_back(xdlops_conv.construction_params[0]);
+
+    result.invoker_factory =
+        MakeWinogradInvokerFactory<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>(
+            ctx, xdlops_conv.invoker_factory.value(), true);
+
+    return result;
+}
+
+template <int WinoDataH, int WinoFilterH, int WinoDataW, int WinoFilterW>
+PerformanceImplicitGemmForwardV4R4Xdlops
+ConvMPBidirectWinograd_xdlops<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::Search(
+    const ConvolutionContext& ctx) const
+{
+    return ConvHipImplicitGemmForwardV4R4Xdlops().Search(GetTransformedConvContext(ctx));
+}
+
+template <int WinoDataH, int WinoFilterH, int WinoDataW, int WinoFilterW>
+template <typename B, typename T>
+int ConvMPBidirectWinograd_xdlops<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::
+    RunAndMeasureSolution(const miopen::Handle& handle,
```

Comment:
I recommend putting this on hold until #203 is merged in. `RunAndMeasureSolution` will not be needed after that.

Reply:
Hmm, wait, it's too long... I'd better remove it .....right now.

Reply:
After #203, all Invokers must be able to measure elapsed time. Please double-check your Invoker and then press `[Resolve conversation]`. Thanks.

Reply:
They were able to do this before #203. 
But now they can make tunning without 'RunAndMeasureSolution'.

But this solver during the tuning process, calls 'ConvHipImplicitGemmForwardV4R4Xdlops.Search()' and tunes ConvHipImplicitGemmForwardV4R4Xdlops solver without using its own kernels.